.circleci/config.yml

```diff
@@ -23,9 +23,16 @@ jobs:
           command: |
             gem install bundler --version="~> 1"
       - run:
-          name: Install PHP
+          name: Update apt
           command: |
             sudo apt-get update
+      - run:
+          name: Install Python
+          command: |
+            sudo apt-get install python
```

Comment:
Since these are single commands, the multi-line syntax can be removed (the `|`). Also I wonder if you might be able to get away with `--no-install-recommends` to speed up CI slightly.

Reply:
fcd2482 (and no recommended installs, so no benefit in this case)